package-wasm.sh

```diff
@@ -17,7 +18,11 @@ mkdir build-wasm
 cd build-wasm
 emcmake cmake .. -DUSE_WASM=1 -DOPENCV_DIR=$CIMBAR_ROOT/opencv4
 make -j5 install
-(cd ../web/ && tar -czvf cimbar.wasm.tar.gz cimbar_js.js cimbar_js.wasm index.html main.js)
+(cd ../web/ && bash wasmgz.sh)
+
+if [ -n $SKIP_JS ]; then
```

Comment:
This is mostly for the automated testing case. But it can be useful for dev as well.